build.properties.dist

```diff
@@ -227,7 +227,7 @@ solr.core.dir = ${solr.dir}/server/solr/${solr.core}
 # -------------------
 
 # Development modules to enable.
-drupal.modules.dev = admin_toolbar config_update field_ui menu_ui views_ui
```

Comment:
Why remove admin toolbar? it is not a part of this ticket and as far as I remember Pieter checked that it is compatible with his keyboard so, accessibility is ok.!